.github/workflows/tests.yml

```diff
@@ -138,8 +132,13 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile --network-concurrency 1
 
-      - name: Run solhint and forge-fmt
+      - name: Run Forge formatter and Solhint
         run: yarn lint:check
 
-      - name: Run lintspec
-        run: yarn lint:natspec
+      - name: Install and run Lintspec
+        uses: beeb/lintspec@v0.6.0
+
+      - name: Install and run Bulloak
+        uses: smol-ninja/bulloak-toolchain@v1
+        with:
+          test-dir: 'test/unit'
```

Comment:
Isnt this missing
`run: yarn lint:natspec` and `run: yarn lint:bulloak` respectively?